### PR TITLE
Cap america

### PR DESCRIPTION
**Pull request by Tarto-4**
## Capstone Pull Request
- Added a functional .html file
- Uploaded background images and social icons
- Added a .css file for webpage styling 
- The added script file contains web features
- Configured linters packages
- Updated README file

### CODE REVIEW

**Review by levy002 — COMMENTED** [review]
## Hello @Tarto-4  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :x: :x: 

You have submitted a project, that doesn't have a professional README file.

- [x] Kindly make sure to include all main sections from the [Microverse README template](https://github.com/microverseinc/readme-template) which will help to explain your repository contents and have a clean and professional Readme. Please make sure to customize it according to your project and fill in the needed information and remember all main sections must be included except the FAQ section which is optional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Tarto-4** [comment]
Thank you for the feedback. I have updated my read me file as instructed.

File: README.md [hunk 1]
```diff
@@ -1 +1,195 @@
...
+To run tests, run the following command:
+
+Example command:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+  npx hint .
+```
+
+### Deployment
```
Comment (BregornOriginal) [inline README.md:140]: Good job updating your README file 💯 

- Kindly as was requested, deploy your application on GitHub Pages and add the link to this section

File: index.html [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                    </div>
+                    <div class="first-group">
+                        <h4 class="fourth-p">&nbsp Amazon</h4>
+                        <h4 class="fifth-p">&nbsp TikTok</h4>
+                    </div>
+                </div>
+            </div><br><br>
+        </section>
+    </main>
+    <footer class="bottom" id="updates">
```
Comment (BregornOriginal) [inline index.html]: - [x] Good work implementing the footer. To follow the same layout in the Behance design, kindly align the text to the right and the logo to the left

- [x] As we can see the footer is bigger on the Behance design, kindly add more padding to your footer on both pages

- [x] Kindly make the background in your `Desktop version white on your home page`

Footer on desktop version home page

image

Footer on the desktop version of the page

image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                    </div>
+                    <div class="first-group">
+                        <h4 class="fourth-p">&nbsp Amazon</h4>
+                        <h4 class="fifth-p">&nbsp TikTok</h4>
+                    </div>
+                </div>
+            </div><br><br>
+        </section>
+    </main>
+    <footer class="bottom">
```
Comment (BregornOriginal) [inline about.html]: - [x] Good work implementing the footer. To follow the same layout in the Behance design, kindly align the text to the right and the logo to the left

File: index.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+                    <p>📍108 Holland Drive, Malibu.</p>
+                </div>
+            </section><br><br><br>
+        </section>
+
+        <section class="program" id="program">
+            <br>
+            <h4 class="text-center section-title-white">Main Program</h4>
+            <hr class="line-name">
+            <div class="program-container container">
```
Comment (BregornOriginal) [inline index.html]: Good job implementing the `Main Program` section. To match better with the Behance design 

- [x] Kindly reduce the gap between each card

- [x] Change the font-family text in this section for `Lato`

- [x] Reduce the size of your description text

- [x] And lastly, as you can see the card container is centered in the middle of the section. You could reduce the space between the cards and the section to center it in the middle, maybe adjusting the size of the container or using margins.

image

image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+            </div>
+            <br>
+            <div class="container text-center">
+                <a href="#" class="ticket-btn p-3" id="ticket">Get Your Ticket Now</a>
+                <a href="#" class="link-to-program p-3">See The Whole Program</a>
+            </div><br><br>
+        </section>
+
+        <section class="first-speaker">
```
Comment (BregornOriginal) [inline index.html]: Excellent work implementing dynamically your Featured Speakers section 💯 

There are some changes to make here to make this a better match with Behances design

- [ ] I can see that your font-size is too big for this section and maybe that it's using all the space on the cards, maybe you can reduce it a bit to use the space for each element

- [ ] Kindly fix the size of the images to have the same size between them. I've seen that you are giving a size in the properties, but maybe the image is different in your assets.

- [ ] As we can see, here the font-family used is `Lato`, kindly implement it

- [ ] There's a bit more space between the grey line and the subtitle and description, kindly add this missing space


image

image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+                    <li><a class="second-nav-i" href="#program">Program</a></li><br>
+                    <li><a class="second-nav-i" href="#join">Join</a></li><br>
+                    <li><a class="second-nav-i" href="#sponsor">Sponsor</a></li><br>
+                    <li><a class="second-nav-i" href="#updates">News</a></li><br>
+                    <li><a href="#" class="mt-3 p-2" id="cc-btn">CC Campaign</a></li><br>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <nav class="mobile-nav">
```
Comment (BregornOriginal) [inline index.html]: Good work implementing the hamburger button! 💯 It works properly

- [ ] To match the Behance design, kindly add a bit more space between the button and the borders

image

File: style.css [hunk 7]
```diff
@@ -0,0 +1,386 @@
...
+    font-weight: 700;
+  }
+
+  .message-p {
+    width: 60%;
+    font-size: 20px;
+    font-weight: 600;
+  }
+
+  .program-container {
```
Comment (BregornOriginal) [inline style.css]: In your program container, there are some changes to make

- [ ] Reducing the size of your font in this section will give you more space in the cards to work and will match also Behance design

- [ ] Kindly align the description of the text to match the rest of the cards

- [ ] Here there's a Lato font family too in the design

image

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Tarto-4  

Good job so far!

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

cat-driving-serious

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by amejid — APPROVED** [review]
Hi @Tarto-4, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
- Changes implemented :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**